evmcore/apply_fake_genesis.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
-var FakeGenesisTime = inter.Timestamp(1608600000 * time.Second)
+var FakeGenesisTime = inter.Timestamp(time.Duration(time.Now().Unix()) * time.Second)
```

Comment:
I am not sure this will go well with Norma, where multiple nodes are initialized using the fake-genesis code independently and are then expected to cooperate with each other. If the time of the fake-genesis creation is used, as listed here, each of those nodes would already be out-of-sync from the start.

Reply:
ok. I'll remove the timestamp changes from this PR so that we can merge thee hash fix and we keep discussing a better way to approach the timestamp issue.